app/assets/stylesheets/nucore.scss

```diff
@@ -206,9 +206,6 @@ th.currency {
 .currency-input input {
   width: 7em;
 }
-.total {
```

Comment:
Ya, it can. In [PR 250](https://github.com/tablexi/nucore-open/pull/250/files#diff-c014e377576c262389d42610b44fed6dR209) I added this bold style because I throught it was a unique selector, but it turns out there are other things using it that I don't want to apply the bold style to.

I updated `app/views/orders/receipt.html.haml` to use the `<strong>` tag instead of this style.